src/3d/qgsrubberband3d.cpp

```diff
@@ -38,51 +38,46 @@
 #include <Qt3DCore/QGeometry>
 #endif
 
+#include "qgs3dutils.h"
+#include "qgsmessagelog.h"
+#include "qgstessellatedpolygongeometry.h"
+
 #include <Qt3DRender/QGeometryRenderer>
 #include <QColor>
 
 
 /// @cond PRIVATE
 
 
-QgsRubberBand3D::QgsRubberBand3D( Qgs3DMapSettings &map, QgsWindow3DEngine *engine, Qt3DCore::QEntity *parentEntity, Qgis::GeometryType geometryType )
+QgsRubberBand3D::QgsRubberBand3D( Qgs3DMapSettings &map, QgsWindow3DEngine *engine, Qt3DCore::QEntity *parentEntity, const Qgis::GeometryType geometryType, const bool isOutlineDashed )
```

Comment:
let's not add `isOutlineDashed` as another argument to the constructor (it's not very important property, and the list of arguments is getting long) - instead let's add `void setMarkerOutlineStyle( Qt::PenStyle style )` setter (+ similar getter), and use `QgsSymbolLayerUtils::encodePenStyle()` to get appropriate string representation for marker symbol.